### DISCUSSION TITLE
Mongodb questions

### DISCUSSION THREAD

**Discussion by HierarchyMan** [diamond-rip/Eden#299] [Q&A] [answered]
I was never really familiar with mongodb
I only know how to get it to work when running locally on my pc
I wanna switch to an online host but idk what to do about mongodb
So how or which host do I use for the mongo database? (Preferably free) what things should it have and stuff and what IP do I put.

**Comment by GoodestEnglish** [comment, marked as answer]
Clevercloud is a popular free mongodb hosting. I use it for plugin testing purposes. The downside is they had a few downtime sometimes.
Here's a toturial from BGHD development about FateUHC mongodb setup, but it is familiar to Eden also https://youtu.be/EwzbShyg00E

**Reply by HierarchyMan** [reply]
Thanks a lot
